src/modelfree/configs/ray/aws.yaml

```diff
@@ -2,19 +2,20 @@ cluster_name: aprl
 min_workers: 0
 max_workers: 10
 initial_workers: 0
-target_utilization_fraction: 0.8
+target_utilization_fraction: 0.4
 idle_timeout_minutes: 5
 
 docker:
   image: humancompatibleai/adversarial_policies:latest
-  container_name: ray
+  container_name: ray-cody
```

Comment:
Good practice to suffix the container name with your username but this should not be committed.

I make a copy of the file, `aws-localcfg.yaml`, which is gitignore'd, and make these changes. Then periodically do a manual diff between them and copy over any changes that are important to `aws.yaml`.

Reply:
Okay